service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/any/flexibleresource/ControlledFlexibleResource.java

```diff
@@ -79,10 +79,12 @@ public <T> T castByEnum(WsmResourceType expectedType) {
 
   // -- getters used in serialization --
 
+  @Override
   public WsmResourceFields getWsmResourceFields() {
     return super.getWsmResourceFields();
   }
 
+  @Override
   public WsmControlledResourceFields getWsmControlledResourceFields() {
     return super.getWsmControlledResourceFields();
   }
```

Comment:
This isn't relevant to the integration test failure, but during my debugging adventure I noticed the other controlled resources had an `@Override` annotation here.